internal/controller/api_to_config_test.go

```diff
@@ -1309,6 +1309,81 @@ func TestConversion(t *testing.T) {
 			},
 			err: nil,
 		},
+		{
+			name: "Single Router and injection",
+			fromK8s: []v1beta1.FRRConfiguration{
+				{
+					Spec: v1beta1.FRRConfigurationSpec{
+						BGP: v1beta1.BGPConfig{
+							Routers: []v1beta1.Router{
+								{
+									ASN: 65001,
+									ID:  "192.0.2.1",
+								},
+							},
+						},
+						Raw: v1beta1.RawConfig{
+							Config: []byte("foo"),
+						},
+					},
+				},
+			},
+			expected: &frr.Config{
+				Routers: []*frr.RouterConfig{
+					{
+						MyASN:        65001,
+						RouterID:     "192.0.2.1",
+						Neighbors:    []*frr.NeighborConfig{},
+						VRF:          "",
+						IPV4Prefixes: []string{},
+						IPV6Prefixes: []string{},
+					},
+				},
+				ExtraConfig: "foo\n",
+			},
+			err: nil,
+		},
+		{
+			name: "Single Router and double injection",
```

Comment:
is there any value in testing a multi line raw config?

Reply:
I can add